omnigibson/__init__.py

```diff
@@ -98,6 +105,8 @@ def create_app():
     global app
     from omni.isaac.kit import SimulationApp
     app = SimulationApp({"headless": gm.HEADLESS})
+    from omni.isaac.core.utils.extensions import enable_extension
+    enable_extension("omni.kit.window.viewport")
```

Comment:
why is this needed?

Reply:
This is isaac sim bug on windows where this extension is not automatically loaded and will crash OG